guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Capsule_Server.adoc

```diff
@@ -4,22 +4,28 @@
 The host name of {SmartProxyServer} is referenced by {ProjectServer} components, and all hosts registered to it.
 This procedure ensures that you update all references to the new host name.
 
-.Prerequisites
-+
+[IMPORTANT]
+====
+* Both the `hostname` and `hostname -f` commands must return the FQDN of {SmartProxyServer} or the `{project-change-hostname}` script will fail to complete.
+
+* If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _<old_fqdn>_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
+====
+
```

Comment:
Two notes one after the other is very poor practice and extremely noisy. 
Please don't do this.

Reply:
Technically, I have no objections. Please work with @melcorr in arranging this in better form.